docs/choreo-extension/release-notes/version-1.0.0.md

```diff
@@ -8,4 +8,43 @@ Another significant enhancement is the ability to log in to the Choreo platform
 
 Furthermore, the extension enables you to create various types of components including RESTful APIs, GraphQL APIs, webhooks, or scheduled apps. With the extension, you can create these new components, implement the necessary logic, and deploy them to Choreo easily from within the VS Code IDE itself.
 
+### Signing In
+
+<img src="./v1.0.0-screens/sign-in.gif" class="cInlineImage-full"/>

```

Comment:
```suggestion
<img src="./v1.0.0-screens/sign-in.gif" class="cInlineImage-full"/>

```
We need to give the absolute (complete) path when adding GIfs or images. This is due to a rendering problem of MKDocs.

Reply:
This should be fine as we are not displaying the release note on the website.